Mathlib/Topology/ContinuousFunction/Algebra.lean

```diff
@@ -118,7 +121,8 @@ theorem nat_cast_apply [NatCast β] (n : ℕ) (x : α) : (n : C(α, β)) x = n :
   rfl
 #align continuous_map.nat_cast_apply ContinuousMap.nat_cast_apply
 
--- ### "int_cast"
+/-! ### "int_cast"-/
```

Comment:
Can we fix (lean4ify) the names of these headers at least if we are going to make them appear in the docs, and put them in backticks

Reply:
agreed